mycroft/configuration/config.py

```diff
@@ -174,7 +175,7 @@ class Configuration:
     __patch = {}  # Patch config that skills can update to override config
 
     @staticmethod
-    def get(configs=None, cache=True):
+    def get(configs=None, cache=True, remote=True):
```

Comment:
So after reviewing the rest of the code, is it because we now have an unknown number of configuration paths courtesy of `xdg.BaseDirectory.load_config_paths()`?

Reply:
I'm not entirely sure anymore to be honest, it has been ages since I've touched this code (hint hint, please don't wait so long before reviewing/accepting community PR's :wink:). Checking the code I don't fully see why this was made an option. We call this function with `remote=False` most of the time but at least 1 place uses `remote=True` (or well, it doesn't pass a value for it which means it defaults to True). I'm guessing I had a reason for that but yes, this was ages ago.

Reply:
i think this was to avoid cyclic import errors due to RemoteConf using `from mycroft.api import is_paired`